src/services/permanence-conseillers/permanence/repositories/permanence-conseiller.repository.js

```diff
@@ -61,9 +61,46 @@ const setPermanence = db => async (permanenceId, permanence, conseillerId, userI
   });
 };
 
+const updatePermanences = db => async permanences => {
+  await permanences.forEach(permanence => {
```

Comment:
humm peut être passer en promesse à cause du for each

Reply:
yes je pense aussi, sinon tu peux faire la boucle for pour éviter de gérer les promesses